leasing/management/commands/set_group_model_permissions.py

```diff
@@ -1182,6 +1182,27 @@
         6: ("view",),
         7: ("view", "add", "change", "delete"),
     },
+    "areasearchattachment": {
+        1: ("view",),
+        2: (
```

Comment:
Just regarding the formatting, what if you remove the last comma? I think it would align better with the rest of the files style, and be maybe more readable overall